roles/nvidia-hpc-sdk/defaults/main.yml

```diff
@@ -1,19 +1,27 @@
 ---
-hpcsdk_version: "2020_207"
+# Version numbers
+hpcsdk_major_version: "20"
+hpcsdk_minor_version: "9"
+hpcsdk_file_cuda: "11.0"
+hpcsdk_default_cuda: "11.0"
```

Comment:
Can you document these a bit, what's the difference?

Reply:
Yes, this is definitely confusing :) I'll add some comments.